File: HELLO PYTHON/day07/myupdate.py
import pymysql
 
conn = pymysql.connect(host='localhost', user='root', password='python',port=3305,
                       db='python', charset='utf8')
 
curs = conn.cursor()
e_id = '3'
e_name = '4'
sex = '4'
addr = '4'

sql = f"""   //낮은버전일경우 안될경우도 있음
update
    emp
set
    e_name = '{e_name}',
    sex = '{sex}',
    addr = '{addr}'
where
    e_id = '{e_id}'

"""
cnt = curs.execute(sql)
print(cnt)
conn.commit()
 
conn.close()

File: HELLO_WEB/src/main/java/kr/co/aiai/dao/MyUpdate.java
package kr.co.aiai.dao;
import java.sql.*;

public class MyUpdate {

	public static void main(String[] args) throws Exception {

		Class.forName("com.mysql.cj.jdbc.Driver");
		Connection conn = DriverManager.getConnection("jdbc:mysql://127.0.0.1:3305/python", "root", "python");
		Statement stmt = conn.createStatement();
		
		String sql = "";
		sql += "update emp ";
		sql += "set ";
		sql += "e_name = '4', ";
		sql += "sex = '4', ";
		sql += "addr = '4' ";
		sql += "where ";
		sql += "e_id = '3' ";

		int cnt = stmt.executeUpdate(sql);
		System.out.println("cnt:"+cnt);

		stmt.close();
		conn.close();
	}
}



